examples/platform_services/android/app/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+apply plugin: 'flutter'
```

Comment:
I'd love to see this updated to use the latest gradle scripts (that I just merged :) ), but that can be in a later change.